<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WynikiPosrednie</class>
 <widget class="QMainWindow" name="WynikiPosrednie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>956</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Informacje o epoce</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(95, 95, 95);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>833</width>
      <height>684</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wybierz satelite</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Satelity"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Uwagi:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Uwagi">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Brak uwag.</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Poprawka czasu satelity[s]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="dSV">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Poprawka relatywistyczna</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="dRealValue">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cofnięcie epoki o czas:[s]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="dPopCzasu">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Azymut [st]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="Azymut">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Kąt elewacji [st]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="Elewacja">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wartość poprawki troposferycznej[m]</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="dTropo">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wartość poprawki jonosferycznej[m]</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="dIono">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pseudoodległości P1[m]</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="P1">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Początkowa pseudoodległości[m]</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="dPoczCxc">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Końcowa pseudoodległość [m]</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="dKoncCxc">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string>Współrzędne na początkową epokę obliczeń</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Współrzędna X :[m]</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="wspPX">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Współrzędna Y:[m]</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="wspPY">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Współrzędna Z: [m]</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="wspPZ">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string>Poprawione współrzędne</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Poprawiony czas obliczeń</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="dPopCzasSV">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Współrzędna X :[m]</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="wspX">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Współrzędna Y:[m]</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="wspY">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Współrzędna Z: [m]</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLabel" name="wspZ">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="P2">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pseudoodległości P2[m]</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>801</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Iteracja</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dx</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dy</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>cDt</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
